test/e2e/e2e.go

```diff
@@ -1325,30 +1345,41 @@ var _ = ginkgo.Describe("e2e non-vxlan external gateway through a gateway pod",
 // functionality to the container loopbacks. To verify traffic reaches the
 // gateways, tcpdump is running on the external gateways and will exit successfully
 // once an ICMP packet is received from the annotated pod in the k8s cluster.
+// Two additional gateways are added to verify the tcp / udp protocols.
+// They run the netexec command, and the pod asks to return their hostname.
```

Comment:
Note for discussion later, but we should start thinking in moving different tests features to individual files, as I did with the multicast.go` per example, this is growing too much ....

Reply:
Agree, how about creating an external_gateway.go in https://github.com/ovn-org/ovn-kubernetes/pull/2029 once this gets in?
That would also solve my issue of where to put some refactored util functions :P

Reply:
I´m not too familiar with the features, but if we can aggregate per feature that will be great